build_runner/lib/src/generate/build_impl.dart

```diff
@@ -341,6 +344,18 @@ class BuildImpl {
     }
   }
 
+  /// Deletes any of `outputs` which previously were output.
```

Comment:
Surround `outputs` with square braces instead of backticks?

Reply:
done